### PR TITLE
[Improve](txn) Add some fuzzy test stub in txn

### PR DESCRIPTION
**Pull request by deardeng**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by dataroaring** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java [hunk 1]
```diff
@@ -171,15 +186,28 @@ public synchronized void run() {
             return;
         }
 
+        // /api/debug_point/add/FE.ALTER_JOB_V2_RUN?stop=true
+        if (DebugPointUtil.isEnable("FE.ALTER_JOB_V2_RUN")) {
```
Comment (yujun777) [inline fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java:190]: boolean stop = DebugPointUtil.getDebugParamOrDefault(FE.ALTER_JOB_V2_RUN, "stop", false)

**Review by yujun777 — CHANGES_REQUESTED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java (hunk 1 above)
Comment (yujun777) [reply fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java:190]: use  if (DebugPointUtil.isEnable("FE.STOP_ALTER_JOB_RUN")) { return; }  ?

each point name should clear ?

File: be/src/olap/task/engine_publish_version_task.cpp [hunk 2]
```diff
@@ -91,6 +92,17 @@ Status EnginePublishVersionTask::finish() {
     int64_t transaction_id = _publish_version_req.transaction_id;
     OlapStopWatch watch;
     VLOG_NOTICE << "begin to process publish version. transaction_id=" << transaction_id;
+    DBUG_EXECUTE_IF("EnginePublishVersionTask.finish", {
```
[the remark this answers is not in the document]
Comment (deardeng) [reply be/src/olap/task/engine_publish_version_task.cpp]: fixed

File: fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java [hunk 3]
```diff
@@ -155,6 +157,19 @@ public void setFinishedTimeMs(long finishedTimeMs) {
         this.finishedTimeMs = finishedTimeMs;
     }
 
+    // /api/debug_point/add/{name}?value=100
+    private void stateWait(final String name) {
+        long waitTimeMs = DebugPointUtil.isEnable(name) ? DebugPointUtil.getDebugParamOrDefault(name, 0) : 0;
```
[the remark this answers is not in the document]
Comment (deardeng) [reply fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java:162]: fixed

File: fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java (hunk 1 above)
Comment (deardeng) [reply fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java:189]: fixed

**Review by yujun777 — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**